src/blueapi/service/model.py

```diff
@@ -54,10 +54,15 @@ class PlanModel(BlueapiBaseModel):
     """
 
     name: str = Field(description="Name of the plan")
+    properties: Dict[str, Any] = Field(description="Plan arguments description")
+    required: List[str] = Field(description="Names of required arguments")
```

Comment:
I think it would be better if these were nested, say inside another object called `PlanSchema`? Makes it easier to build client code that uses the schema if it's isolated in one place.